docs/examples/tcloud_getting_started.md

```diff
@@ -0,0 +1,238 @@
+# Tobiko Cloud: Getting Started
+
+Tobiko Cloud is a data platform that extends SQLMesh to make it easy to manage data at scale. We're here to make it easy to get started and feel confident that everything is working as expected. After you've completed the steps below, you'll have achieved the following:
+
+- Login to Tobiko Cloud via the browser
+- Connect Tobiko Cloud to your local machine via the CLI
+- Connect Tobiko Cloud to your data warehouse
+- Verify Tobiko Cloud interacts with your data warehouse as expected
+
+## Prerequisites
+
+Onboarding requires the below steps from the Tobiko team to be completed before Tobiko Cloud can be used.
+  
+The Solutions Architect will:
+
+- Set up a 1 hour meeting with you to fully onboard
+- Request a new Tobiko Cloud account will be created for you (single tenant by default)
+- Share a temporary password link that expires in 7 days
+- Make sure you save the password in your own password manager
+
+You will need to:
+
+- Ensure you have data warehouse admin rights to update permissions and create new users with create/update/delete permissions on a specific database (ex: `database.schema.table`)
+
+For SQLMesh (open source) to Tobiko Cloud Migrations Only:
+
+- Solutions Architect will send you a script to extract your current state to send to the Tobiko Cloud engineers to validate before migration
+- After testing, Solutions Architect will schedule a migration date and call to move your state to Tobiko Cloud. There will be some downtime if you are running SQLMesh in a production environment.
+
+> Note: if you must be on VPN to access your data warehouse or have specific security requirements, please let us know and we can discuss options to ensure Tobiko Cloud can securely connect.
+
+Technical Requirements:
+
+- Python 3.10+
```

Comment:
With releases starting sometime next week it will be 3.9+ again. So probably best to just say 3.9+ here.

Reply:
will update